### ISSUE TITLE
Palisade region shading

### ISSUE THREAD

**Issue by chrisboyle** [chrisboyle/sgtpuzzles#385]
Suggested by Kevin Buzzard in an email of 16 Feb:
> For Palisade the issue for me is not lines on/off, I would like to be
> able to shade regions. Again this would involve a major programming
> exercise. I've mentioned this already for Loopy. There's an Android
> implemention of Loopy which colours the _squares_ blue or yellow
> depending on whether they're known to be inside or outside the loop.
> For Signpost there is some of this "local colouring" when internal
> runs are formed -- this would be great for Palisade too (basically
> colour two squares the same colour when the line between them is
> removed and then let coloured regions grow accordingly). It's a far
> more visual way of understanding what's going on and ultimately would
> make it easier for the end user I suspect (rather than my extremely
> obscure suggestion above).

**Comment by SilverMagpie** [comment]
I agree; even something as simple as filling in a finished area similar to Filling would be helpful. It can be hard to tell if I've closed an area at a glance.
